infomaniak-build-tools/windows/Readme.md

```diff
@@ -363,7 +363,7 @@ In the Solution Explorer, go to the available view:
 
 Select CMake targets.
 
-Right-click on the Kdrive (executable) >> Debug >> Kdrive.exe (bin\kDrive.exe).
+Right-click on the kDrive (executable) >> Debug >> kDrive.exe (bin\kDrive.exe).
 
 Once Kdrive.exe is running, right-click on the Kdrive_client (executable) >> Debug >> Kdrive_client.exe (bin\kDrive.exe).
```

Comment:
Are these names correct? (I mean "kDrive" versus "Kdrive").

Reply:
I corrected it to "kDrive" in the last rev.